packages/@azure/servicebus/data-plane/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@azure/service-bus",
+  "author": "Microsoft Corporation",
+  "version": "0.1.0",
+  "license": "MIT",
+  "description": "Azure ServiceBus SDK for JS",
+  "homepage": "http://github.com/azure/azure-sdk-for-js",
```

Comment:
@AlexGhiondea - It would be nice to have the url all the way upto the data plane folder

https://github.com/Azure/azure-sdk-for-js/tree/master/packages/%40azure/servicebus/data-plane.

Folks use that link to reach the source code location on github.

Reply:
@amarzavery Yes, we made that change in a separate commit: https://github.com/Azure/azure-sdk-for-js/blob/master/packages/%40azure/servicebus/data-plane/package.json#L7

And welcome back!